lib/msf/core/handler/reverse/comm.rb

```diff
@@ -39,9 +39,8 @@ def select_comm
     comm
   end
 
-  def via_string_for_ip(ip, comm)
+  def via_string(comm)
     comm_used = comm
-    comm_used ||= Rex::Socket::SwitchBoard.best_comm(ip)
     comm_used ||= Rex::Socket::Comm::Local
```

Comment:
Not sure if this is the best option given that according to https://github.com/rapid7/rex-socket/blob/d959b513bb8c52637d8e37e62852e97f746be80e/lib/rex/socket/switch_board.rb#L122-L128 this code should be finding the best possible com and only if it can't find it, then it should default to `Rex::Socket::Com::Local`.

That being said I can see the argument for this given that right now there appears to only be one subclass under `Rex::Socket::Com` which is `Local`, however for expansion purposes down the line this suggestion kind of removes the flexibility we may want to add in the future.

Reply:
After further internal discussion, this seems to be fine.